README.md

```diff
@@ -51,5 +71,13 @@ recipes:
     - '  x'
 ```
 
-If you need to find a Slimefun item ID you can join the [official SF Discord here](https://slimefun.dev/discord) and doing `!item <name/ID>` in the `#bot-spam` channel.
+### :computer: Discord
+If you need to find a Slimefun item ID you can join the official Slimefun Discord by clicking the image below this text and doing `!item <name/ID>` in the `#bot-spam` channel.
+You can also generally chat about LiteXpansion in the `#litexpansion` channel.
+
```

Comment:
Maybe point them towards the #litexpansion channel aswell

Reply:
I literally wrote that in the section you marked... "You can also generally chat about LiteXpansion in the '#litexpansion' channel."

Reply:
How tf am i this blind